auth-server->auth-mailer depends on l10n module that is extraneous and would be remove by prune

Happened to notice this. Not sure what exactly to do about it. But thought I should document.

If you do a clean install of auth-server, and do `npm ls`, it reports:
`npm ERR! extraneous: fxa-content-server-l10n@0.0.0`.

If you then do `npm prune`, tests fail, since `node_modules/fxa-auth-mailer/translator.js` wants to load `node_modules/fxa-auth-mailer/node_modules/fxa-content-server-l10n` files. I'm not really sure why npm thinks this is extraneous, since there is not other fxa-content-server-l10n under node_modules that I can see.

--- a/.gitignore
+++ b/.gitignore
@@ -18,3 +18,4 @@ secret*
 npm-debug.log
 server.pot
 .mail_output
+fxa-content-server-l10n/

--- a/grunttasks/l10n-extract.js
+++ b/grunttasks/l10n-extract.js
@@ -17,7 +17,7 @@ module.exports = function (grunt) {
       files: [{
         expand: true,
         flatten: true,
-        cwd: path.join(pkgroot, 'node_modules', 'fxa-content-server-l10n', 'locale', 'templates', 'LC_MESSAGES'),
+        cwd: path.join(pkgroot, 'fxa-content-server-l10n', 'locale', 'templates', 'LC_MESSAGES'),
         dest: pkgroot,
         src: [
           'server.pot'

--- a/lib/senders/translator.js
+++ b/lib/senders/translator.js
@@ -21,7 +21,7 @@ function parseLocale(locale) {
   return poParseFile(
     path.join(
       __dirname,
-      '../../node_modules/fxa-content-server-l10n/locale',
+      '../../fxa-content-server-l10n/locale',
       i18n.normalizeLocale(locale),
       'LC_MESSAGES/server.po'
     ),

--- a/scripts/download_l10n.sh
+++ b/scripts/download_l10n.sh
@@ -9,8 +9,13 @@ if [ -z "$FXA_L10N_SHA" ]; then
     FXA_L10N_SHA="master"
 fi
 
-DOWNLOAD_PATH="https://github.com/mozilla/fxa-content-server-l10n.git -b $FXA_L10N_SHA"
+rm -rf fxa-content-server-l10n
+
+DOWNLOAD_PATH="https://github.com/mozilla/fxa-content-server-l10n.git"
 
 echo "Downloading L10N files from $DOWNLOAD_PATH..."
 # Download L10N using git
-git clone --depth=10 $DOWNLOAD_PATH
+git clone --depth=20 $DOWNLOAD_PATH
+cd fxa-content-server-l10n
+git checkout $FXA_L10N_SHA
+cd ..

--- a/scripts/download_l10n.sh
+++ b/scripts/download_l10n.sh
@@ -9,8 +9,8 @@ if [ -z "$FXA_L10N_SHA" ]; then
     FXA_L10N_SHA="master"
 fi
 
-DOWNLOAD_PATH="git://github.com/mozilla/fxa-content-server-l10n.git#$FXA_L10N_SHA"
+DOWNLOAD_PATH="https://github.com/mozilla/fxa-content-server-l10n.git -b $FXA_L10N_SHA"
 
 echo "Downloading L10N files from $DOWNLOAD_PATH..."
-# Download L10N using npm
-npm install $DOWNLOAD_PATH
+# Download L10N using git
+git clone --depth=10 $DOWNLOAD_PATH

